Orange/canvas/application/settings.py

```diff
@@ -359,6 +360,24 @@ def __setupUi(self):
 
         tab.setLayout(form)
 
+        # Error Reporting Tab
+        tab = QWidget()
+        self.addTab(tab, self.tr("Error Reporting"),
+                    toolTip="Settings related to error reporting")
+
+        class MachineIDLineEdit(QLineEdit):
+            def __init__(self):
+                super().__init__()
+                self.setText(get_machine_id())
+
+            def focusOutEvent(self, QFocusEvent):
+                super().focusOutEvent(QFocusEvent)
+                set_machine_id(self.text())
+
+        form = QFormLayout()
+        form.addRow("Machine ID:", MachineIDLineEdit())
+        tab.setLayout(form)
```

Comment:
The settings stored by QSettings() do not reset between versions (in fact they persist since Orange 2.7).